lib/landrush/util/dnsmasq.rb

```diff
@@ -0,0 +1,10 @@
+module Landrush
+  module Util
+    class Dnsmasq
+      def self.nm_managed?
+        nm_config = Pathname('/etc/NetworkManager/NetworkManager.conf')
+        File.exist?(nm_config) && File.readlines(nm_config).grep(/^dns=dnsmasq$/).any?
```

Comment:
@jmontleon regarding this check for 'dns=dnsmasq' - is dnsmasq something you enabled manually (maybe for other purposes) prior to using Landrush or is this enabled by default on your Fedora box? Which version of Fedora are you using btw.

And if dnsmasq is not enabled for default, should we be enabling via NetworkManager by effectively adding _dns=dnsmasq_? See also issue #260?